package.json

```diff
@@ -75,6 +75,7 @@
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint sass-lint",
+    "package": "cd addon && npm run package",
```

Comment:
Same as the change for package.json for the addon - not directly related to this PR.